addons/website/static/src/snippets/s_floating_blocks/floating_blocks.edit.js

```diff
@@ -1,13 +0,0 @@
-import { FloatingBlocks } from "./floating_blocks";
-import { registry } from "@web/core/registry";
-
-const FloatingBlocksEdit = I => class extends I {
-    callToAction() {}
-};
-
-registry
-    .category("public.interactions.edit")
-    .add("website.floating_blocks", {
-        Interaction: FloatingBlocks,
-        mixin: FloatingBlocksEdit,
-    });
```

Comment:
In fact, this was useless: I supposed there was a callToAction function you wanted to cancel but it does not exist at all